bridge/contracts/nftbridge/NFTBridge.sol

```diff
@@ -312,6 +313,10 @@ contract NFTBridge is
     IERC721Metadata metadataIERC = IERC721Metadata(tokenAddress);
     string memory tokenURI = metadataIERC.tokenURI(tokenId);
 
+    if (originalTokenAddressBySideTokenAddress[tokenAddress] != NULL_ADDRESS) {
+      ERC721Burnable(tokenAddress).burn(tokenId);
+    }
+
     emit Cross(
       tokenAddress,
```

Comment:
The emited event should be the originalTokenAddress on the main chain https://github.com/rsksmart/tokenbridge/blob/master/bridge/contracts/Bridge.sol#L426

Reply:
> The emited event should be the originalTokenAddress on the main chain https://github.com/rsksmart/tokenbridge/blob/master/bridge/contracts/Bridge.sol#L426

Shouldn't we change the name of the first param of the Cross event to 'originalTokenAddress'?

Reply:
I think we should to avoid confusion, @rafaiovlabs  are you ok if we merge this and then rename it on your branch?

Reply:
Sure no problem. Regards.